<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
border-image:url(:/img/fondo.png)
}
padding: 50%;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="horizontalWidget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">width:100px;
border-image:url(:/img/logo.png)
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">  width: 50px; /* Diámetro del botón */
  height: 50px; /* Diámetro del botón (mismo valor que el ancho) */
  background-color: rgba(255, 255, 255, 0.7); /* Color blanco con opacidad del 70% */
  border: 2px solid #000; /* Borde de 2px sólido */
  border-radius: 50%; /* Hace que el botón sea circular */
  display: flex;
  justify-content: center;
  align-items: center;
  cursor: pointer;
  font-weight: bold;
  text-decoration: none;</string>
        </property>
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;
color:white</string>
        </property>
        <property name="text">
         <string>Host: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;
color:white</string>
        </property>
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;
color:white</string>
        </property>
        <property name="text">
         <string>Conexion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="check" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">width:100px;
border-image:url(:/img/check.png)
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color:green</string>
        </property>
        <property name="text">
         <string>Exitosa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">width:100px;
border-image:url(:/img/error.png)
</string>
        </property>
        <layout class="QHBoxLayout" name="error"/>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>Algo salio mal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>190</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="styleSheet">
          <string notr="true">color:white;
font: 20pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>TITULO</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_DDL">
           <property name="styleSheet">
            <string notr="true">color:white;
font: 14pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>DML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_DML">
           <property name="styleSheet">
            <string notr="true">color:white;
font: 14pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>DML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_DCL">
           <property name="styleSheet">
            <string notr="true">color:white;
font: 14pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>DML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonPlay">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
border-image:url(:/img/play.png)
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonReset">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
border-image:url(:/img/reset.png)
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;
color:white</string>
         </property>
         <property name="text">
          <string>Ingresa tu consulta MySql:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_input">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Arial&quot;;
color:white</string>
         </property>
         <property name="text">
          <string>Salida:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_output">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhMultiLine</set>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;RESULTADO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>189</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imgs/images.qrc"/>
 </resources>
 <connections/>
</ui>
